server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/AbstractGroupUpdate.java

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
don't you think it would be better using composition instead inheritance in this case ? The class is providing utility methods to check/validate parameters, to write groups and share common constants. Maybe create 2 classes, GroupUpdateValidator and UserGroupJsonWriter ? Whereas for the constants, I'm not sure what should be done. WDYT ?